### PR TITLE
Update readme.md to include important info

### LINKED ISSUE #70: [21/10/2022]: Add a detailed README .md file to the dev_team2 branch
#### Description

- *Update the README.md to include important information*

#### Which task are you working on?

- [ ] *Addresses one of the tasks stipulate in issue #10 *

### What needs to be done?

- [ ] *Update the README with relevant information to new contributors*
- [ ] *Update the readme to include important information about the repository*

### Who want to work on this issue?

I would like to be assigned to work on this issue

### PR DESCRIPTION
**Pull request by JulianaSau**
### Summary

Fixes #70
Addresses #10 

*Updating README.md with more information about the project and repository and include information important to new contributors*

### List of changes proposed in this PR (pull-request)

* *Added a few badges to track stars, contributors, forks*
* *Added the figma design link*
* *Added the getting started section to enable new contributors to understand how to set up either the frontend or the backend for development*

### Screenshots
image
image
image
image
image
image
image

### What should a reviewer concentrate their feedback on?

- [ ] *If anything should be added or removed*
- [ ] Everything looks ok?

### CODE REVIEW

**Comment by JulianaSau** [comment]
Hey @sandygudie, please review my pr

File: README.md [hunk 1]
```diff
@@ -1,16 +1,160 @@
...
+```bash
+yarn build
+```
+
+## Things to note as you contribute
+
+- Any page you code has to be responsive.
+- Write clear and concise commit messages.
+- Always run `yarn` after pulling code from the upstream repository.
+- Always, **Always** make pull requests to the **dev** branch, not the main branch. All pull requests to the main branch will not be merged.
```
Comment (sandygudie) [inline README.md:124]: Welldone @JulianaSau 
...."to the dev_team2 branch"...
please

File: README.md [hunk 2]
```diff
@@ -1,16 +1,160 @@
...
+
+### STARTING THE SERVER
+
+- To start the whole application server
+
+```bash
+yarn dev
+```
+
+- Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```
Comment (sandygudie) [inline README.md:110]: The Vite app default port is 5173, it does not open at 3000 (unless customized)

**Review by sandygudie — CHANGES_REQUESTED** [review]

**Comment by JulianaSau** [comment]
Thanks for the review. I've updated the pr with the requested changes

**Review by sandygudie — APPROVED** [review]

**Review by sandygudie — APPROVED** [review]

**Comment by sandygudie** [comment]
@BatoolMM this is ready for merging
Also, Please, if it's possible; I would like to request merge privilege for the MOOCs Frontend development so that development could be much faster. Thanks

**Comment by BatoolMM** [comment]
@all-contributors please add @sandygudie for mentoring

**Comment by BatoolMM** [comment]
Just saw this - you now have merge privileges for the MOOCs Frontend development.

**Pull request merged**